lib/auth/index.js

```diff
@@ -53,6 +53,8 @@ authRouter.get('/logout', function (req, res) {
   if (config.debug && req.isAuthenticated()) {
     logger.debug('user logout: ' + req.user.id)
   }
-  req.logout()
+  req.logout(()=>{
+    logger.debug('Please implement logout callback here.')
+  })
```

Comment:
It should be like:

```suggestion
  req.logout((err) => {
    if (err) { return next(err) }
    res.redirect(config.serverURL + '/')
  })
```

and add `next` as the last parameter for get route callback function

Reply:
Hi @hcyuser,
Please consider this suggestion.